app/controllers/arrival/ArrivalXmlNegativeAcknowledgementController.scala

```diff
@@ -44,14 +44,11 @@ class XmlNegativeAcknowledgementController @Inject()(
     implicit request =>
       arrivalMessageService.getXMLSubmissionNegativeAcknowledgementMessage(arrivalId).flatMap {
         case Some(rejectionMessage) =>
-          val json = Json.obj("nctsEnquiries" -> frontendAppConfig.nctsEnquiriesUrl, "functionalError" -> rejectionMessage.error)
+          val json = Json.obj("contactUrl" -> frontendAppConfig.nctsEnquiriesUrl, "functionalError" -> rejectionMessage.error)
 
           renderer.render("xmlNegativeAcknowledgement.njk", json).map(Ok(_))
         case _ =>
-          renderer.render("internalServerError.njk").map {
-            content =>
-              InternalServerError(content)
-          }
+          renderer.render("internalServerError.njk").map(InternalServerError(_))
```

Comment:
Do we need to pass in the `nctsEnquiries` url?  looks like the view for technical difficulties takes it, but its not being passed in.